src/components/form-helpers/form-helpers.js

```diff
@@ -86,6 +107,26 @@ const _PhoneFieldItem = ({
     }
   };
 
+  const validatePhone = (value) => {
```

Comment:
Maybe you could move this logic into `validations.js`? as the rest of the validations.

Reply:
Yes, but in this case, it is coupled to `iti` object, I could pass it to `validations.js`, but I am not sure, this validation is different to the rest validations. what do you think?

Reply:
What about sending this `iti` as a parameter?. I think it would be more clear to have all validations in one place.

Reply:
Yes, it is possible, let me continue thinking about it. Thanks!